eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -31,91 +33,70 @@ jobs:
+      CleanTargetFolder: true
+
   - task: DotNetCoreCLI@2
     displayName: Publish $(functionApp) app
     inputs:
       command: publish
       publishWebProjects: false
       zipAfterPublish: false
       modifyOutputPath: false
-      projects: '$(Build.ArtifactStagingDirectory)\BenchmarkApps\DotNet\$(functionApp)\HelloHttp.csproj'
+      projects: '$(Build.ArtifactStagingDirectory)/PerformanceTestApps/$(functionApp)/HelloHttp.csproj'
       arguments: -c Release -o $(functionAppOutputPath) -f net9.0
-      workingDirectory: $(Build.ArtifactStagingDirectory)\BenchmarkApps\DotNet\$(functionApp)
+      workingDirectory: $(Build.ArtifactStagingDirectory)/PerformanceTestApps/$(functionApp)
 
-  - script: dotnet tool install -g Microsoft.Crank.Controller  --version "0.2.0-*"
-    displayName: Install Microsoft.Crank.Controller 
+  - script: dotnet tool install -g Microsoft.Crank.Controller --version "0.2.0-*"
+    displayName: Install Microsoft.Crank.Controller
 
   - task: PowerShell@2
     displayName: Run crank-controller
     inputs:
       targetType: 'inline'
       script: |
-        $workerRuntime = "$(functionsWorkerRuntime)"
-        $functionAppOutputPath = "$(functionAppOutputPath)"
-        $configFilePath = ".\..\host\eng\perf\http.benchmarks.yml"
-        $hostLocation = ".\..\..\..\host"
-
-        $crankArgs = "--config  $configFilePath --scenario hellohttp --profile win2022 --load.options.reuseBuild true --description `"$(runDescription)`" --command-line-property --no-metadata --no-measurements --json $(benchmarkResultsJsonPath) --property sourceVersion=$(sourceVersion) --property buildNumber=$(buildNumber) --property buildId=$(buildId) --variable FunctionsWorkerRuntime=$workerRuntime --variable HostLocation=$hostLocation --variable FunctionAppPath=$functionAppOutputPath"
-        if ($env:CRANK_SQL_CONNECTION_STRING) {
-          $crankArgs += " --table HttpBenchmarks --sql CRANK_SQL_CONNECTION_STRING"
-        }
+        $crankArgs = "--config $(configFilePath) --scenario hellohttp --profile win2022 --load.options.reuseBuild true --description `"$(runDescription)`" --command-line-property --no-metadata --no-measurements --json $(benchmarkResultsJsonPath) --property sourceVersion=$(sourceVersion) --property buildNumber=$(buildNumber) --property buildId=$(buildId) --variable FunctionsWorkerRuntime=$(functionsWorkerRuntime) --variable HostLocation=$(hostLocation) --variable FunctionAppPath=$(functionAppOutputPath)"
         $crankArgs += " ${{ parameters.additionalCrankArgs }}"
         $command = "crank $crankArgs"
+
+        if ('${{ parameters.storeResultsInDatabase }}' -eq 'true') {
+            $command += " --table HttpBenchmarks --sql `"$(BenchmarkResultsSqlConnectionString)`""
+        }
+
         Write-Host "Running command: $command"
         Invoke-Expression $command
 
   - task: PowerShell@2
-    displayName: Output logs
+    displayName: Functions host logs
```

Comment:
You mentioned this is a log stream. This implementation will get all the logs in one request. Do you foresee this ever being too large for a single request?

Reply:
The endpoint we call is converting the log to a rolling log (limit to last X entries). we can certainly revisit if we run into issues with large logs.